src/Libraries/RevitNodes/Elements/AdaptiveComponent.cs

```diff
@@ -475,7 +273,23 @@ public static AdaptiveComponent ByParametersOnCurveReference(double[] parameters
                 throw new ArgumentNullException("familyType");
             }
 
-            return new AdaptiveComponent(parameters, ElementCurveReference.TryGetCurveReference(revitCurve).InternalReference, familyType);
+            var result = GeometryObjectSelector.ByReferenceStableRepresentation(
+                        revitCurve.InternalReference.ConvertToStableRepresentation(Document));
+            var curve = result as Autodesk.DesignScript.Geometry.Curve;
+            if (curve == null)
+            {
+                var curves = result as List<Autodesk.DesignScript.Geometry.Curve>;
+                if (curves != null && curves.Count() > 0)
+                {
+                    curve = curves[0];
```

Comment:
What if selected curve reference is exported as multiple curves? Can we throw exception?